docs/integration_resources/resources/usd_tutorials/usd_tutorial_04.md

```diff
@@ -0,0 +1,79 @@
+## Converting Between Layer Formats
+
+This tutorial is focused on converting layer files between the different native USD file formats using the operators from the Shift_USD catalog.
+
+The base file for the example, `Sphere.usd`, can be downloaded from the [OpenUSD](https://github.com/PixarAnimationStudios/OpenUSD) Github project repository, from `extras/usd/tutorials/convertingLayerFormats/` folder.
+
+
+### Converting between .usda/.usdc and .usd
+
+1. Open Shift application. A new window's instance of Shift should appear like the following one:
+
+<figure>
+    <img src="images/usd_tutorial_04/step_01_t04.png" alt="Shift UI.">
+    <figcaption><b>Figure 1</b>: Shift UI.</figcaption>
+</figure>
+
+2. First, add a new **<tt>UsdStageOpen</tt>** node to the board and rename it as `OpenUsdFile`. Using the **Inspector** widget, set as the value for the **filepath** input plug a filepath pointing to a `.usd` file. For the current example, the filepath can be: `D:/USD_TUTORIALS/04/Sphere.usd`. The board should look like the following:
+
+<figure>
+    <img src="images/usd_tutorial_04/step_02_t04.png" alt="Added 'OpenUsdFile' node to the board.">
+    <figcaption><b>Figure 2</b>: Added 'OpenUsdFile' node to the board.</figcaption>
+</figure>
+
+3. To save a `.usda` file, add a new **<tt>UsdStageSave</tt>** node to the board and rename it as `SaveAsUsdaFile`. Connect the **stage** output plug from the `OpenUsdFile` node to the **stage** input plug of the new node. Set as the value for the **filepath** input plug a valid filepath to a `.usda` file, for example, `D:/USD_TUTORIALS/04/Sphere.usda`. The board should look like the following:
```

Comment:
Can you check the file for similar sentences? The "for example" sounds a bit unnatural in that sentence structure. "Such as" makes the sentence flow better, I don't know how to explain it haha.

```suggestion
3. To save a `.usda` file, add a new **<tt>UsdStageSave</tt>** node to the board and rename it as `SaveAsUsdaFile`. Connect the **stage** output plug from the `OpenUsdFile` node to the **stage** input plug of the new node. Set the filepath input plug to a valid .usda file path, such as D:/USD_TUTORIALS/04/Sphere.usda. The board should look like the following:
```